default_ocp_versions.json

```diff
@@ -1,8 +1,14 @@
 {
 	"4.6": {
-		"release_image": "quay.io/openshift-release-dev/ocp-release:4.6.8-x86_64"
+		"display_name": "4.6.8",
+		"release_image": "quay.io/openshift-release-dev/ocp-release:4.6.8-x86_64",
+		"rhcos_image": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso",
+		"support_level": "production"
 	},
 	"4.7": {
-		"release_image": "quay.io/openshift-release-dev/ocp-release@sha256:2419f9cd3ea9bd114764855653012e305ade2527210d332bfdd6dbdae538bd66"
+		"display_name": "4.7-pre-release",
+		"release_image": "quay.io/openshift-release-dev/ocp-release@sha256:2419f9cd3ea9bd114764855653012e305ade2527210d332bfdd6dbdae538bd66",
+		"rhcos_image": "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso",
+		"support_level": "beta"
 	}
```

Comment:
@ErezAlster We are all set on metrics right? (We should always have the openshift version)

Reply:
yes, @oshercc took care for it :)